README.rst

```diff
@@ -179,6 +186,62 @@ $ curl https://httpbin.org/get?show\_env=1
       "url": "http://httpbin.org/get?show_env=1"
     }
 
+$ curl -XPOST https://httpbin.org/file/dir1/dir2?name=myfile.json
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+::
+
+    {
+      "file_path": "/tmp/dir1/dir2/myfile.json"
```

Comment:
is this where we will store the file within tmp? if its going to be available to the world, maybe strip that out and just show as if everything is being read from prefix on?  if the user relies on this as the prefix path above, i'd assume its incorrect.